### PR TITLE
zh: install/operator/index

### PR DESCRIPTION
**Pull request by youhonglian**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
cc @kebe7jun

**Comment by kebe7jun** [comment]
Istio 需要大写，在 ID 里面也是。

**Review by kebe7jun — APPROVED** [review]

File: content/zh/docs/setup/install/operator/index.md [hunk 1]
```diff
@@ -8,7 +8,11 @@ test: no
 status: Beta
 ---
 
-除了手动安装、升级、和卸载 Istio，您还可以用 Istio [Operator](https://kubernetes.io/zh/docs/concepts/extend-kubernetes/operator/) 管理安装。
+{{< warning >}}
+不鼓励在新的 Istio 安装中使用操作符，以支持 [Istioctl](/zh/docs/setup/install/istioctl) 和 [Helm](/zh/docs/setup/install/helm) 安装方法。虽然运营商将继续得到支持，新功能请求不会被优先考虑。
```
[the remark this answers is not in the document]
Comment (zirain) [reply content/zh/docs/setup/install/operator/index.md:12]: Operator是k8s的扩展模式，不能直接翻译为 `操作符`

[the remark this answers is not in the document]
Comment (youhonglian) [reply content/zh/docs/setup/install/operator/index.md]: 改为 `Operator` 了

**Review by zirain — APPROVED** [review]

**Pull request merged**